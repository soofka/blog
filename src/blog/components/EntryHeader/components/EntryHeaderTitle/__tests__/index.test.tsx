import { shallow } from 'enzyme';
import * as React from 'react';

import { EntryHeaderTitle } from '../';

describe('<EntryHeaderTitle/>', () => {

  it('renders correctly', () => {
    expect(shallow(
      <EntryHeaderTitle
        title="testTitle"
        link="testTitleLink"
      />,
    )).toMatchSnapshot();
  });

});
