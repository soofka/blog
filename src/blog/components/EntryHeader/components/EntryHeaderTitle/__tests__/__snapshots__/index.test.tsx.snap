// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EntryHeaderTitle/> renders correctly 1`] = `
<styled.h1>
  <Link
    replace={false}
    to="testTitleLink"
  >
    testTitle
  </Link>
</styled.h1>
`;
